mainwindow.cpp

```diff
@@ -157,17 +157,17 @@ MainWindow::MainWindow(QSettings &settings)
     // TODO: Are this checks are enough?
     // TODO: Claim on mkdir fails.
     // TODO: Do not do it on start up. Choose other good time.
-    if (!QDir(QDir::home().filePath(".john")).exists()) {
-        QDir::home().mkdir(".john");
+    if (!QDir(QDir::home().filePath("john")).exists()) {
```

Comment:
On Mon, May 11, 2015 at 06:08:29AM -0700, math07 wrote:
> > @@ -157,17 +157,17 @@ MainWindow::MainWindow(QSettings &settings)
> >      // TODO: Are this checks are enough?
> >      // TODO: Claim on mkdir fails.
> >      // TODO: Do not do it on start up. Choose other good time.
> > -    if (!QDir(QDir::home().filePath(".john")).exists()) {
> > -        QDir::home().mkdir(".john");
> > +    if (!QDir(QDir::home().filePath("john")).exists()) {
> 
> Removinh the . in .john is temporary, it's until we get a final decision from Solar because right now dots no longer works with cvs repo version

You may want to use "_" instead of "." not just removing the char.
Thanks!

-- 
Regards,
Aleksey Cherepanov